src/lib/server/index.js

```diff
@@ -0,0 +1,31 @@
+import express from "express";
+import logger from "../logger";
+import { filename } from "../logsColorScheme";
+import requestHandler from "./requestHandler";
+
+const isProduction = process.env.NODE_ENV === "production";
+//@TODO: read host from config file
+const host = "localhost";
+//@TODO: understand why different port numbers between prod and dev
```

Comment:
Awesome, thanks for the explanation @toddw! I'll remove my todo comment for this now. This was actually an internal comment to myself that I accidentally left, but glad I got something out of it 😄